src/ewmh_m2m/screen.py

```diff
@@ -28,8 +28,11 @@ def get_sibling_screens(current: Geometry, screens: Iterable[Geometry]) -> Dict[
 
 
 def get_sibling_screen(siblings: Dict[Ordinal, List[Geometry]],
-                       direction: Ordinal,
+                       direction: Optional[Ordinal],
                        no_wrap: bool) -> Optional[Geometry]:
+    if not direction:
+        return (get_sibling_screen(siblings, Ordinal.EAST, no_wrap=False)
+                or get_sibling_screen(siblings, Ordinal.SOUTH, no_wrap=False))
```

Comment:
I don’t agree with this change. It is too deeply nested inside the code. I think it’s better that you handle that in your bash script (as you proposed: `m2m -d EAST || m2m -d SOUTH`)
```suggestion
                       direction: Ordinal,
                       no_wrap: bool) -> Optional[Geometry]:
```